api/v1beta1/experiment_types.go

```diff
@@ -180,10 +180,10 @@ const (
 type PatchTemplate struct {
 	// The patch type, one of: strategic|merge|json, default: strategic
 	Type PatchType `json:"type,omitempty"`
-	// A Go Template that evaluates to valid patch
-	Patch string `json:"patch"`
 	// Direct reference to the object the patch should be applied to
 	TargetRef *corev1.ObjectReference `json:"targetRef,omitempty"`
+	// A Go Template that evaluates to valid patch
+	Patch string `json:"patch"`
```

Comment:
wonder what triggered the ordering here? 
( not a blocker, just curious )

Reply:
The field ordering controls the serialization ordering now, and putting the target ref above the patch looks better.